gui_config.json

```diff
@@ -19,5 +19,8 @@
     // Enter a number between 0 (not inclusive) and 1.
     "STARTUP_WIDTH_RATIO": 0.75,
     "STARTUP_HEIGHT_RATIO": 0.7,
-    "DISPLAY_ADVANCED_MENU_MESSAGES_IN_TERMINAL": "False"
+    "DISPLAY_ADVANCED_MENU_MESSAGES_IN_TERMINAL": "False",
+    // The maximum difference in pixel spacing between two images of the global group, such that they may be considered to be the same size.
+    // If they must be exactly the same instead of approximately the same, set this to 0.0.
+    "GROUP_MAX_SPACING_DIFF": 0.0001
```

Comment:
This should be added to CLI as well

Reply:
There aren't groups/isn't batch processing in the CLI unless someone changed something and I don't remember it.

Reply:
![image](https://github.com/NIRALUser/NeuroRuler/assets/69875698/a40a3be4-626f-4a30-baa9-9e6dc8f98ce3)

I forgor